### ISSUE TITLE
[BAN]

### ISSUE THREAD

**Issue by shernandezsantana** [austinhuang0131/barinsta#251]
## Answer honestly. All boxes must be checked for this report to be considered.

- [ ] My app is on the latest version available on GitHub or F-Droid. I understand that any other version is not supported.
- [x ] I certify that all actions I have performed on the app constitute human behaviour. I am using the app responsibly and in a way identical to using the official app. Specifically, I did not perform botting or automated key clicks.
- [x ] I have considered other possible reasons for my ban, and I cannot find any substantial claim other than the usage of this app.

## Answer honestly. Check accordingly to your situation.

- [ ] I had prior rule violations on Instagram, specifically:
- [ ] I have admitted the use of InstaGrabber on Instagram.
- [ ] I have admitted the use of InstaGrabber to a friend who uses Instagram.
- [ ] I have modified the source code of the app that I use, other than what is present in this repo. Specifically:

## Describe your case, including your usage pattern, but without private information.

I used Instagrabber on Thursday and tested it a bit, via watching posts and stories. I tested the function of downloading, which was not really.downloading more than a white image. I tested this function on posts and stories.

## Provide your communication with Instagram, without private information of you.

They banned my account temporarily, and reviewed it for 1 day. Then, they decided to ban it permanently without telling me why. I have tried to contact them and appeal, but I cannot appeal properly with their form because I cannot sign in. I never experienced any banning, or warning before. I have tried to explained what happened to me, and I think there are only few explanations. One that I was hacked as well, and other that the app I used was old and did sth weird, which IG cannot differentiate between collecting information weirdly for misuse or someone just watching instagram in a different app.

**Comment by austinhuang0131** [comment]
Did you register your account very recently?

**Comment by shernandezsantana** [comment]
No!! It was created in 2018 and I lost it. I used to document my lifestyle almost daily via stories and weekly via posts.

**Comment by austinhuang0131** [comment]
Sorry, was a bit hurried. Lemme get this straight:

1. You used your account for 2 years and one day ("Thursday") decided to try out InstaGrabber (which version?).
2. Then they "reviewed it" for a day (don't they ask you to verify?) and made the ban permanent.
3. You ticked "I have admitted the use of InstaGrabber _on Instagram_, after the ban." so how? Did you tell Instagram about using this app???

**Comment by shernandezsantana** [comment]
1. I think it was a previous version, but I uninstalled it and now I dont which version. I could download stories.
2. They asked me to verify it and I did verify right away after the temporarily disable.
3. I didnt know I was not supposed to. I am new to this open source stuff. I send an email right before contacting here. I am sorry about this, but I didnt know what to do.

**Comment by austinhuang0131** [comment]
1. So 18.x or 19.x. Noted.
2. Okay.
3. No, Facebook will not allow any other client to access Facebook/Instagram, and no, IG only provides an API for commercial clients to do analytics. What we do is reverse engineering of the official mobile app and provide an alternative client, and we fight against the [walled garden principle](https://en.wikipedia.org/wiki/Closed_platform). While I'm sorry for your loss, if you indeed have explicitly admitted to usage of any non-official client to Instagram directly, then you're basically screwed.

**Comment by shernandezsantana** [comment]
Okey. I can live with my account gone. I hope I can recover my data, though. I think I wont repeat this mistake.
